src/raw_logging.cc

```diff
@@ -59,7 +59,7 @@
 # include <unistd.h>
 #endif
 
-#if defined(HAVE_SYSCALL_H) || defined(HAVE_SYS_SYSCALL_H)
+#if !defined(ANDROID) && (defined(HAVE_SYSCALL_H) || defined(HAVE_SYS_SYSCALL_H))
```

Comment:
andoird-1 has a header "sys/glibc-syscalls.h" which contains "SYS_write" definition, but it has been deprecated a long time ago.
newer platforms does not have this header.